### ISSUE TITLE
URL on click showing 'Security error '

### ISSUE THREAD

**Issue by singhrohan62** [anshumanv/evolutions-webapp#42]
##### This issue is about {Issue Title}

- [ ] I ensure that this issue is not a duplicate of any current open issues(s).
- [ ] Assign this issue to me (I want to work on this).
- [ ] I will send my PR only after I'm assigned to this issue by any maintainer.
- [ ] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.

**Comment by singhrohan62** [comment]
I think that happened 'coz Google Safe browsing detected phishing on wallpup.com (ie, the website from where bg-url has been fetched). Kindly see to it @anshumanv

**Comment by anshumanv** [comment]
Thanks for reporting this, I'll see, you can fix this perhaps if you like.

**Comment by singhrohan62** [comment]
Hey @anshumanv  check the PR I have send, made the necessary changes

**Comment by anshumanv** [comment]
Thanks, I'll merge it tomorrow.
